src/03-control-flow.js

```diff
@@ -146,7 +195,9 @@ function calculateLetterGrade(grade) {}
  *  animalCounts("pig", 1);
  *  //> "1 pig"
  */
-function animalCounts(animal, numberOfAnimals) {}
+function animalCounts(animal, numberOfAnimals) {
+  return numberOfAnimals > 1 ? `${numberOfAnimals} ${animal}s` : `1 ${animal}`;
```

Comment:
Although this solution passes the test, you should extend it to one more case: what would happen if there were 0 animals? Re-configuring your solution a bit will solve that easily!